### DISCUSSION TITLE
Can anyone guide me on how to use confusion matrix method?

### DISCUSSION THREAD

**Discussion by balaganeshmohan** [open-mmlab/mmaction2#901] [Q&A]
I see CM implemented on mmaction/core/evaluation/accuracy.py alone with mean_class_accuracy and topk_accuracy, but when I include confusion_matrix option on '--eval' argument, I get KeyError: 'metric confusion_matrix is not supported'.

**Comment by antbertrand** [comment]
The confusion matrix is solely used to compute the mean_class_accuracy. If you really want to see it during your training, you could simply print it [there](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/core/evaluation/accuracy.py#L79).
